src/game.ts

```diff
@@ -255,16 +260,18 @@ const buildCollateralInput = (txHash: string, txIx: number) => {
 };
 
 const encodeRedeemer = (cmd: Cmd): string => {
-  return Data.to([
+  return Data.to(
     new Constr(1, [
-      new Constr(0, [
-        BigInt(cmd.forwardMove),
-        BigInt(cmd.sideMove),
-        BigInt(0),
-        [],
-      ]),
+      [
```

Comment:
Should: update the redeemer decoding (one function below) accordingly.

Reply:
Good catch, fixed